app/controllers/password_resets_controller.rb

```diff
@@ -1,5 +1,55 @@
 class PasswordResetsController < ApplicationController
+  before_action :load_user, only: [:edit, :update]
+
   def new; end
 
+  def create
+    @user = User.find_by email: params[:password_reset][:email].downcase
+    if @user
+      @user.create_reset_digest
+      @user.send_password_reset_email
+      flash[:info] = t "email_reset"
+      redirect_to root_url
+    else
+      flash.now[:danger] = t "email_not_found"
+      render :new
+    end
+  end
+
   def edit; end
+
+  def update
+    if params[:user][:password].empty?
+      @user.errors.add :password, t("cant_be_empty")
```

Comment:
I think no need to please this business logic here. 
And previously in the User model you already used this logic(Presence validation for the :password)